### DISCUSSION TITLE
#30DaysOfServerless core concepts link is broken

### DISCUSSION THREAD

**Discussion by jaspreet-commits** [Azure/Cloud-Native#11] [Serverless September: Share Your Feedback]
Hello team ,
The links to core concept is broken in serverless page.can you please check it

**Comment by nitya** [comment]
Hi there - thanks for letting us know. Will update it - for now all those links will point to the central blog page, but once each post is published, I will update these to reflect a direct link to the post. Thanks so much for letting us know!

**Comment by nitya** [comment]
Locking discussion - moved to issues instead - please continue updates there.
https://github.com/Azure/Cloud-Native/issues/12
